dist/ami/files/scylla_install_ami

```diff
@@ -89,33 +88,11 @@ if __name__ == '__main__':
     run('/opt/scylladb/scripts/scylla_setup --ntp-domain amazon --no-coredump-setup --no-sysconfig-setup --no-raid-setup --no-io-setup --no-bootparam-setup --no-ec2-check')
     run('/opt/scylladb/scripts/scylla_sysconfig_setup --ami')
     run('/opt/scylladb/scripts/scylla_bootparam_setup --ami')
-    os.remove('/home/centos/.ssh/authorized_keys')
+    os.remove('/home/ec2-user/.ssh/authorized_keys')
```

Comment:
Let's create a new user scyllaadm and add an alias centos for it. This way users can still use centos, and we can teach new users to use scyllaadm.

The build scripts can still use ec2-user.

We'll need to make the cloud-init stuff that copies the keys work for both ec2-user and scyllaadm (or perhaps we can make ec2-user an alias of scyllaadm).

Reply:
Added scyllaadm as default user and add an alias centos, also deleted ec2-user since it no longer needed.
https://github.com/scylladb/scylla/pull/5540/files#diff-2fe20e870f705ad288eafddbd5031182R90
https://github.com/scylladb/scylla/pull/5540/files#diff-e9b6b9a258c40b82ef4794df2cf0821eR1